packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md

```diff
@@ -185,9 +186,17 @@ The timeline graph shows use over the last week. When a rate violation occurs, y
 
 ##### Bar graph
 
-The **Top Requests by IP** bar graph shows the top 10 IP addresses that are consuming the most traffic for the API. This view enables you to check if there is a single IP or multiple IPs that are creating the spike in traffic.
+The bar graph shows either the top 10 IP addresses or top 10 API Tokens that are consuming the most traffic for the API. This view enables you to check if there are IP addresses or API Tokens that are creating the spike in traffic.
+
+Display the desired bar graph by selecting **Top Requests by IP** or **Top Requests by API Token** in the dropdown menu.
 
-Click an IP address bar once to view the traffic for that IP address on the line graph. Double-click any IP address bar to view the traffic for all the IP addresses on the line graph.
+Click an IP address bar or API Token once to view the traffic for that IP address or API Token on the line graph. Double-click any IP address bar or API Token to view the traffic for all the IP addresses or Tokens on the line graph.
```

Comment:
1. Should we move bar after API Token?  
2. I know you did not add this text originally but I think the reference to double-click is confusing because it will show "total request" traffic initially even without clicking.  Double-clicking just shows and hides the filtered line so it is unnecessary.

```suggestion
Click an IP address or API Token bar once to view the traffic for that IP address or API Token on the line graph. 
```

Reply:
1. Yes, done. 2. Will remove, thanks. I'll update both in a local commit.